Added automatic language detection to Azure Transcription Service (#4239)
opencast/opencast: docs/guides/admin/docs/workflowoperationhandlers/microsoft-azure-start-transcription-woh.md

|source-tag|The flavor of the file to be sent for translation.|EMPTY|transcript|
|skip-if-flavor-exists|If this flavor already exists in the media package, skip this operation.<br/>To be used when the media package already has a transcript file. Optional|false|captions/vtt+en-us|
|language-code|The language code to use for the transcription. Optional. If set, it will override the configuration language code|EMPTY|en-US, [supported languages](https://docs.microsoft.com/de-de/azure/cognitive-services/speech-service/language-support?tabs=speechtotext#speech-to-text)|

> |auto-detect-language|Activate automatic language detection by Azure. Optional. Overrides the language set in language-code. If set, will override the value in the configuration|false|true

Reviewer: ```suggestion
|auto-detect-language|Activate automatic language detection by Azure. Optional. Overrides the language set in \`language-code\`. If set, will override the value in the configuration|false|true
```

Unsure of the correct Markdown, but `language-code` should look different than normal text here.

> |auto-detect-languages|A list of language codes. The Azure language auto detection will pick it's detected language from one of these. The language auto detection cannot detect any languages not specified in this list. The list needs to have at least one element and can have at most four elements.|EMPTY|en-US,de-DE,it-IT

Reviewer: ```suggestion
|auto-detect-languages|A list of language codes. Restricts Azure's language autodetection to this list of languages. The list needs to have at least one element and can have at most four elements.|EMPTY|en-US,de-DE,it-IT
```

Author: I don't agree with this rewording. To me, "restricts" implies that the autodetection would be "unrestricted" if `auto-detect-languages` is not specified, which is incorrect. If no languages are specified, the auto detection (and thus, the workflow operation) will fail. (...unless there is a list defined in the global config I believe)

How about:
"A list of language codes. MUST be set if not set already in configuration and will override configuration. The Azure language auto detection chooses from the given list and cannot detect any language not in the given list. The list needs to have at least one element and can have at most four elements."

Reviewer: Ok, I misunderstood what this config item meant.  Yours is better.  What about "MUST be set if not set already in **service** configuration"?  I was confused which configuration you meant, and I can see adopters being confused as well.
